packages/medusa/src/api-v2/admin/customer-groups/[id]/customers/batch/route.ts

```diff
@@ -0,0 +1,66 @@
+import { AdminCustomerGroupResponse, BatchMethodRequest } from "@medusajs/types"
```

Comment:
@olivermrbl it's a pretty small change in 1 place, so I'd rather do it sooner than later. I am actually planning to just unify all of the batch endpoints in 1 go, both on FE and BE as it will be faster